### PR TITLE
[CON-245] Add Network Monitoring Flag

### PR DESCRIPTION
**Pull request by jonaylor89**

### CODE REVIEW

File: audius-cli [hunk 1]
```diff
@@ -147,10 +147,10 @@ def health_check(ctx, service):
 @cli.command()
 @click.option("-y", "--yes", is_flag=True)
 @click.option("--seed", is_flag=True)
+@click.option("--network-monitoring", is_flag=True)
 @click.option("--chain", is_flag=True)
 @click.argument("service", type=service_type)
-@click.pass_context
```
Comment (cheran-senthil) [inline audius-cli:152]: The code does not work without this

Comment (jonaylor89) [reply audius-cli:152]: oops, that was a bad merge

File: discovery-provider/docker-compose.yml [hunk 2]
```diff
@@ -109,6 +109,21 @@ services:
...
+    ports:
+      - 9091:9091
+    networks:
+      - discovery-provider-network
+  
+  network-monitoring:
+    image: audius/network_monitoring
+    container_name: network_monitoring
+    depends_on:
+      - network-monitoring-db
```
Comment (cheran-senthil) [inline discovery-provider/docker-compose.yml:124]: This does not exist 👀

File: discovery-provider/docker-compose.yml [hunk 3]
```diff
@@ -136,8 +151,6 @@ services:
       - "30300:30300"
       - "30300:30300/udp"
       - "8545:8545"
-    networks:
-      - discovery-provider-network
```
Comment (cheran-senthil) [inline discovery-provider/docker-compose.yml:140]: Why did you remove this?

File: discovery-provider/docker-compose.yml (hunk 2 above)
Comment (jonaylor89) [reply discovery-provider/docker-compose.yml:124]: I know, that's why I wanna hop on a call - this is still a draft PR

File: discovery-provider/docker-compose.yml (hunk 3 above)
Comment (jonaylor89) [reply discovery-provider/docker-compose.yml:140]: mrege error - will fix

Comment (jonaylor89) [reply discovery-provider/docker-compose.yml]: fixed

**Comment by SidSethi** [comment]
no reason to ship this anytime soon - doesn't seem like priority and adds complexity to discovery provider infra that we likely don't need

@jonaylor89 how do you feel about dropping this work for now? network monitoring seems plenty stable as is

**Comment by jonaylor89** [comment]
> no reason to ship this anytime soon - doesn't seem like priority and adds complexity to discovery provider infra that we likely don't need
> 
> @jonaylor89 how do you feel about dropping this work for now? network monitoring seems plenty stable as is

This PR is basically done, it's just blocked by [this PR](https://github.com/AudiusProject/audius-protocol/pull/3431) which builds and pushes the network monitoring container. This work isn't super time consuming and I'm totally fine with keeping it as a timeboxed, side thing since it hasn't seemed to get in the way of normal work at all.

**Comment by SidSethi** [comment]
closing this since not actively being worked on, and work is tracked in linear
